includes/admin/settings/class-settings-gateways.php

```diff
@@ -479,6 +513,101 @@ public function render_enabled_gateways($field, $settings)
+                                fill="#F2CC0C"></path>
+                        </svg>
+                        <?php
+                        esc_html_e('What\'s new', 'give'); ?>
+                    </div>
+                    <?php
+                    esc_html_e(
+                        'GiveWP 3.0 introduces an enhanced forms experience powered by the new Visual Donations Form Builder. However, we are still working on gateway compatibility with the new forms experience.',
+                        'give'
+                    ); ?>
+                    <?php
+                    if ($supportedGateways) : ?>
+                        <div class="give-payment-gateway-settings-dialog__content-title"><?php
+                            esc_html_e('Supported gateways', 'give'); ?></div>
+                        <div class="give-payment-gateway-settings-dialog__content-itemsContainer">
+                            <?php
+                            foreach ($supportedGateways as $gateway) : ?>
+                                <div class="give-payment-gateway-settings-dialog__content-item">
+                                    <svg width="16" height="16" viewBox="0 0 16 16" fill="none"
+                                         xmlns="http://www.w3.org/2000/svg">
+                                        <path fill-rule="evenodd" clip-rule="evenodd"
+                                              d="M7.063.986a1.531 1.531 0 0 1 1.872 0l.783.601.98-.129c.69-.09 1.354.294 1.62.935l.377.913.911.376h.002c.641.267 1.025.93.935 1.62l-.13.98.602.783a1.534 1.534 0 0 1 0 1.872l-.601.783.129.98c.09.69-.294 1.354-.935 1.62h-.002l-.91.377-.378.912a1.537 1.537 0 0 1-1.62.936l-.98-.13-.783.601a1.531 1.531 0 0 1-1.872 0l-.782-.6-.98.129a1.537 1.537 0 0 1-1.62-.936l-.377-.912-.911-.376H2.39a1.537 1.537 0 0 1-.935-1.621l.129-.98-.601-.783a1.533 1.533 0 0 1 0-1.872l.601-.782-.129-.98c-.09-.69.294-1.354.935-1.62l.002-.001.91-.376.377-.913a1.537 1.537 0 0 1 1.62-.935l.98.13.783-.602zm3.741 5.82a.667.667 0 0 0-.943-.943L7.333 8.392 6.47 7.53a.667.667 0 1 0-.943.943L6.86 9.806c.26.26.683.26.943 0l3-3z"
+                                              fill="#459948"></path>
+                                    </svg>
+                                    <?php
+                                    echo $gateway; ?>
+                                </div>
+                            <?php
+                            endforeach; ?>
+                        </div>
+                    <?php
+                    endif; ?>
+                    <button class="give-payment-gateway-settings-dialog__content-button"><?php
+                        esc_html_e('Got it', 'give'); ?></button>
+                    <a href="#" class="give-payment-gateway-settings-dialog__content-link"><?php
```

Comment:
Do we have a shortlink for this?

Reply:
Also if its an external link: ```rel="noopener noreferrer``` https://lw.slack.com/archives/C02QJ8F07B3/p1668790561688639

Reply:
Updated on f9634e927d486bd69e099338d4f50b5ef205b3d1